crates/walrus-service/src/node/config.rs

```diff
@@ -594,6 +598,33 @@ impl BlobRecoveryConfig {
     }
 }
 
+/// Configuration for the pending sliver cache.
+#[serde_as]
+#[derive(Debug, Clone, Deserialize, Serialize, PartialEq, Eq)]
+#[serde(default)]
+pub struct PendingSliverCacheConfig {
+    /// Maximum number of slivers retained in memory while the blob registration is pending.
+    pub max_cached_slivers: usize,
+    /// Maximum number of bytes retained in memory while the blob registration is pending.
+    pub max_cached_bytes: usize,
```

Comment:
Nit: Can you mention that this counts for both metadata and slivers?

Reply:
OK, makes sense. This is not really clear from the parameter name or the docstring though.